domains/mihai.json

```diff
@@ -0,0 +1,11 @@
+{
+  "repo": "https://github.com/8xu/web",
+  "owner": {
+    "username": "8xu",
+    "email": "mihai@8xu.tech",
+    "twitter": "reallymihai"
+  },
+  "record": {
+    "CNAME": "https://8xu.github.io/web/"
```

Comment:
a `CNAME` cannot have a `/`. You can rename the repository to `8xu.github.io`, and then change the `CNAME` to `8xu.github.io`, or if you really want to keep that repository, you can change it to a `URL` record.